tests/src/test/scala/com/nvidia/spark/rapids/GpuCoalesceBatchesSuite.scala

```diff
@@ -134,22 +135,29 @@ class GpuCoalesceBatchesSuite extends SparkQueryCompareTestSuite {
       val df2 = df
         .sort(df.col("longs"))
 
-      val coalesce = df2.queryExecution.executedPlan
+      // execute the plan
+      ExecutionPlanCaptureCallback.startCapture()
+      df2.collect()
+
+      val executedPlan = ExecutionPlanCaptureCallback.getResultWithTimeout() match {
```

Comment:
Should there be a utility method that extracts an executed plan from the callback?  This code block keeps popping up in a number of places.

Reply:
Done.